moon/bulk_test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# NASA qualification round number
+ROUND=3
+
+# after how long should the simulation give up (in seconds); note the round is allowed to run for up to 45 minutes
+TIMEOUT=1800
+
+# desired score
+SCORE="13"
+
+RANGE_START=0
+RANGE_END=49
+
+while getopts "r:t:s:f:l:h" opt; do
+    case ${opt} in
+        h)
+            echo "Usage:"
+            echo " -r <qualification round>"
+            echo " -t <timeout in secs>"
+            echo " -s <desired score>"
+            echo " -f <first seed range>"
+            echo " -l <last seed range>"
+            exit 0
+            ;;
+        r)
+            ROUND=$OPTARG
+            ;;
+        t)
+            TIMEOUT=$OPTARG
+            ;;
+        s)
+            SCORE=$OPTARG
+            ;;
+        f)
+            RANGE_START=$OPTARG
+            ;;
+        l)
+            RANGE_END=$OPTARG
+            ;;
+    esac
+done
+
+
+for i in $(seq $RANGE_START $RANGE_END)
+do
+
+    echo "==========" `date` "====== seed:" $i
+    $HOME/space-challenge/srcp2-competitors/docker/scripts/launch/roslaunch_docker -q -n --run-round $ROUND -s $i >& /dev/null
+
+    # wait 10 secs to start
+    sleep 5
```

Comment:
on my machine the simulation starts much slower - do you have some check that it is running?

Reply:
No, the idea was to add a generous enough delay rather than spending time researching how to test for ROS being operational and ready. We can increase, a few additional seconds don't matter once it starts being used for true bulk testing which is meant to last many hours. Maybe indicate how much time it takes for your docker to start, we add a few more seconds to that and call it a day.

Reply:
5s was a problem for me ... it is running with 50s now (I guessed that NASA container starts simulation and then it continues, so it is just extra buffer, but it seems that it was still starting ...

Reply:
> add a few more seconds to that and call it a day.

we had this in subt and it has bitten us several times really badly... just saying